command-line-interface.md

```diff
@@ -0,0 +1,189 @@
+# Operations
+
+A conformant [runtime][] MUST provide an executable (called `funC` in the following examples).
```

Comment:
On Thu, Jul 28, 2016 at 01:47:34PM -0700, Julian Friedman wrote:
> How about wording like this: "In order to be certified as compliant
> with the spec a certification tool MAY require that a runtime
> implement an approved API specification, so that the testing tool
> can invoke commands using it and validate output. The current list
> of approved API specifications includes only the CLI Spec" (or
> similar)

This MAY (vs. the current MUST) opens the door to having an
OCI-certified runtime implementation that another caller cannot call
(because the API that was certified is not specified).  If we want to
open that door now (and I'd rather not, but I'm not as anti-shim as
@mikebrow), I'd prefer:

  In order to be [certified][], compliant runtimes MUST implement at
  least one of the following APIs:

  * The [command line API](command-line-api.md)

  This allows the certification tool and other portable
  runtime-callers to invoke the runtime in a predicable manner.

  [certified]: home-page of Trademark Board's certification project

And we can add other APIs to runtime-spec as folks see the need (which
I hope will be never ;).